draco/tests/test_soft.py

```diff
@@ -332,8 +329,681 @@ def test_same_field():
     )
 
 
-def test_type_linear():
-    b = soft.blocks["type_linear"]
+def test_same_field_grt3():
```

Comment:
Do you think these tests are good at actually finding potential issues? I think we need to write the test cases in a way where they can reasonably fail. I'll add comments below.

Reply:
The main reason I suggest it is because the file is getting pretty long and I worry about maintainability. 

I'm open to ideas.